src/layouts/Main/components/Footer/Footer.js

```diff
@@ -2,11 +2,12 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import { makeStyles } from '@material-ui/styles';
-import { Typography, Link } from '@material-ui/core';
+import { Typography, Link, FormHelperText } from '@material-ui/core';
```

Comment:
'FormHelperText' is defined but never used. Please delete it.

Reply:
Done.